validphys2/src/validphys/tests/test_pseudodata.py

```diff
@@ -74,11 +72,17 @@ def test_no_savepseudodata():
 def test_read_matches_recreate():
     reads = API.read_fit_pseudodata(fit=PSEUDODATA_FIT)
     recreates = API.recreate_fit_pseudodata(fit=PSEUDODATA_FIT)
-    for read, recreate in zip(reads, recreates):
+    recreate_pdf = API.recreate_fit_pseudodata(fit=PSEUDODATA_FIT)
```

Comment:
What's the difference between this and the previous line?

Reply:
ups, it should be `recreate_pdf_pseudodata`